maintainers/maintainer-list.nix

```diff
@@ -7952,6 +7952,12 @@
     githubId = 668926;
     name = "Maximilian Güntner";
   };
+  mh = {
```

Comment:
Please put the `maintainers/maintainer-list.nix` edit in a separate commit with commit message `maintainers: add mh`. (So there are exactly 2 commits in this PR).